README.md

```
@@ -950,7 +1149,46 @@ you can centralize your logic and reduce duplication of code as you complete the
+task is still in recents, Android restores the original intent into `onCreate` and delivers the new
+one to `onNewIntent`.
+
+<details open>
+   <summary>Kotlin</summary>
+
+   ```kotlin
+   override fun onNewIntent(intent: Intent?) {
+       super.onNewIntent(intent)
+       Klaviyo.getKlaviyoDeepLink(intent)?.let { handleDeepLink(it) }
+   }
+   ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)